hr_time/hr_time_management/doctype/worklog/worklog.json

```diff
@@ -0,0 +1,114 @@
+{
+ "actions": [],
+ "allow_rename": 1,
+ "autoname": "WL-.#####",
+ "creation": "2024-09-08 17:57:24.428560",
+ "doctype": "DocType",
+ "engine": "InnoDB",
+ "field_order": [
+  "employee",
+  "log_time",
+  "task_desc",
+  "amended_from",
+  "task"
+ ],
+ "fields": [
+  {
+   "fieldname": "amended_from",
+   "fieldtype": "Link",
+   "label": "Amended From",
+   "no_copy": 1,
+   "options": "Worklog",
+   "print_hide": 1,
+   "read_only": 1,
+   "search_index": 1
+  },
+  {
+   "fieldname": "employee",
+   "fieldtype": "Link",
+   "in_list_view": 1,
+   "label": "Employee",
+   "options": "Employee",
+   "read_only": 1
+  },
+  {
+   "fieldname": "log_time",
```

Comment:
This field is currently not displayed when users create new worklogs.
However, it should also be possible to create worklogs for previous days.

Reply:
Understood. I did not know that users should be able to log for past time as well.